frontend/src/metabase/visualizations/visualizations/Text/Text.jsx

```diff
@@ -18,10 +19,10 @@ import {
 } from "./Text.styled";
 
 const getSettingsStyle = settings => ({
-  "align-center": settings["text.align_horizontal"] === "center",
-  "align-end": settings["text.align_horizontal"] === "right",
-  "justify-center": settings["text.align_vertical"] === "middle",
-  "justify-end": settings["text.align_vertical"] === "bottom",
+  [CS.alignCenter]: settings["text.align_horizontal"] === "center",
+  [CS.alignStart]: settings["text.align_horizontal"] === "right",
```

Comment:
Related to #46078 it looks like this should be `[CS.alignEnd]` instead? I'll leave a comment in that issue as well - happy to put a PR In for a fix but if it is something minor and it's quicker for someone on the team to do, thought I'd just leave a comment for it.